crates/hc_sandbox/src/cli.rs

```diff
@@ -18,6 +18,10 @@ const DEFAULT_APP_ID: &str = "test-app";
 pub struct HcSandbox {
     #[structopt(subcommand)]
     command: HcSandboxSubcommand,
+    /// Instead of the normal "interactive" passphrase mode,
+    /// collect the passphrase by reading stdin to the end.
+    #[structopt(long)]
+    piped: bool,
```

Comment:
```suggestion
    passphrase_pipe: bool,
    /// Instead of the normal "interactive" passphrase mode,
    /// collect the passphrase from the environment variable HC_PASSPHRASE
    #[structopt(long)]
    passphrase_env: bool,
```

Reply:
i'd also be fine dropping the pipe functionality altogether, as in my personal experience it's cumbersome and i'd prefer an environment variable.